### ISSUE TITLE
Not working on BigSur .

### ISSUE THREAD

**Issue by White7561** [alessandro893/MacOS-WLAN-Driver#24]
So yeah in my limited time using BigSur on my external drive. I saw that it doesn't work. I have SIP disabled. Confirmed via terminal. And the app doesn't patch it. I think I know what the problem is tho. It's that the /System file path etc is different now on BigSur... 

Tried on su mount -uw / . But it failed
![20201114_214406](https://user-images.githubusercontent.com/40349017/99150407-fb835380-26c6-11eb-888e-21a5846b3350.jpg)

Also a quick note. I noticed this after I'm done with reformatting the BigSur temporary drive ... 

On my Macbook Air 2014, usually the wifi doesn't immediately catch the region / CC settings, AKA it needs to catch an AP with that CC settings, then it'll change...

But here on big Sur it immediately uses my country CC . Which is Indonesia. And it's weird since I don't remember having any AP near that broadcasts that CC at that time. I could be wrong tho.   And interestingly, I didn't even log in my apple id there. So the only way it catches my CC is thru the regional setting on setting.. shame I already formatted it. If not I'd try changing the region and seeing if it changes the current WiFi CC...

**Comment by alessandro893** [comment]
You need to run: "csrutil authenticated-root disable" -  in Recovery mode terminal, to be able write to root. But patched kext don't work in my case, it causes boot loop. I'm still working on it.

**Comment by White7561** [comment]
Yeah just checked it out .. turns out you need to "bless" it etc to make sure it can boot... Anyways, thanks for the reply!
